src/components/posts/PostsPage.js

```diff
@@ -39,29 +40,36 @@ const PostsPage = ({ role }) => {
 
   return (
     <>
-      <section className='posts-section'>
-        {postResponse.status}
-        <PostForm handleSubmit={createPost} handleChange={handleChange} inputValue={post.content} />
-        <ul className="posts-list">
-          {posts.map((post, index) => (
-            <li key={index} className='post-item'>
-              <Box>
-                <div className="post-content">{post.content}</div>
-                <Stack className="names-date" spacing={2} direction="row">
-                  <Link to={`/user/${post.user.id}`} className='post-author'>
-                    <Box className="fullname" variant='contained'>
-                      <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
+      <div className='home-container'>
+        <section className='posts-section'>
+          {postResponse.status}
+          <PostForm
+            handleSubmit={createPost}
+            handleChange={handleChange}
+            inputValue={post.content}
+          />
+          <ul className='posts-list'>
+            {posts.map((post, index) => (
+              <li key={index} className='post-item'>
+                <Box>
+                  <div className='post-content'>{post.content}</div>
+                  <Stack className='names-date' spacing={2} direction='row'>
+                    <Link to={`/user/${post.user.id}`} className='post-author'>
+                      <Box className='fullname' variant='contained'>
+                        <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
+                      </Box>
+                    </Link>
+                    <Box className='date-time' variant='contained'>
+                      {dateTimetoRelativeTime(post.createdAt)}
                     </Box>
-                  </Link>
-                  <Box className="date-time" variant="contained">
-                    {dateTimetoRelativeTime(post.createdAt)}
-                  </Box>
-                </Stack>
-              </Box>
-            </li>
-          ))}
-        </ul>
-      </section>
+                  </Stack>
+                </Box>
+              </li>
+            ))}
+          </ul>
+        </section>
+        {localStorage.getItem('role') === 'TEACHER' && <CohortList />}
```

Comment:
As discussed, I think there's a potentially wider refactoring to remove the dependency on local storage and have this passed down using props - but that can be done as a seperate task.